### PR TITLE
DVC-8630 [feat]: add dvc identity update command

### PR DESCRIPTION
**Pull request by kaushalkapasi**
- add command to update identity used for self-targeting functionality

### CODE REVIEW

**Comment by kaushalkapasi** [comment]
Current dependencies on/for this PR:
* main
  * **PR #268** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/cli/268" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/cli/268?utm_source=stack-comment).

File: src/commands/identity/update.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        if (!confirmation) {
+            this.writer.printCurrentValues(
+                `No DevCycle Identity changes made for project: ${this.projectKey} and User ID: ${existingUserId}`
+            )
+            return
+        }
+
+        if (!flags.userId) {
+            const input = await inquirer.prompt([userIdPrompt])
+            dvcUserId = input.userId
```
Comment (chantalchung) [inline src/commands/identity/update.ts:62]: I think we should add or indicate some kind of warning if a user enters a blank dvc id? I was able to just enter and it wiped by id (and all overrides)

File: src/commands/identity/update.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        let dvcUserId
+        let confirmation = false
+        if (existingUserId) {
+            const { confirmUpdate } = await inquirer.prompt([{
+                name: 'confirmUpdate',
+                message: `
+                    A User ID (${existingUserId}) is associated with your account for this project
+                    (${this.projectKey}). Would you like to update it?
+                `,
```
Comment (chantalchung) [inline src/commands/identity/update.ts:45]: are we expecting this prompt to show up in the middle with styling like this?
image

Comment (kaushalkapasi) [reply src/commands/identity/update.ts:45]: I'll fix the formatting, that's a bug!

**Review by chantalchung — APPROVED** [review]

**Pull request merged**